### PR TITLE
Fix crash in FIRInstanceID

### LINKED ISSUE #3439: FIRInstanceID notifyAPNSTokenIsSet -> NSInvalidArgumentException
### [REQUIRED] Step 2: Describe your environment

  * Xcode version: 10.2.1
  * Firebase SDK version: _____
  * Firebase Component: FIRInstanceID (Auth, Core, Database, Firestore, Messaging, Storage, etc)
  * Component version: 6.5.0

### [REQUIRED] Step 3: Describe the problem

We're seeing some crashes in the wild, with stack trace:
```
Fatal Exception: NSInvalidArgumentException
0  CoreFoundation                 0x219c34518 __exceptionPreprocess
1  libobjc.A.dylib                0x218e0f9f8 objc_exception_throw
2  CoreFoundation                 0x219badf8c _CFArgv
3  CoreFoundation                 0x219b36168 -[__NSPlaceholderDictionary initWithObjects:forKeys:count:]
4  CoreFoundation                 0x219b26d44 +[NSDictionary dictionaryWithObjects:forKeys:count:]
5  OurApp                       0x100d44064 __38-[FIRInstanceID notifyAPNSTokenIsSet:]_block_invoke + 1016 (FIRInstanceID.m:1016)
6  libdispatch.dylib              0x219674a38 _dispatch_call_block_and_release
7  libdispatch.dylib              0x2196757d4 _dispatch_client_callout
8  libdispatch.dylib              0x219623004 _dispatch_main_queue_callback_4CF$VARIANT$mp
9  CoreFoundation                 0x219bc5ec0 __CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__
10 CoreFoundation                 0x219bc0df8 __CFRunLoopRun
11 CoreFoundation                 0x219bc0354 CFRunLoopRunSpecific
12 GraphicsServices               0x21bdc079c GSEventRunModal
13 UIKitCore                      0x246177b68 UIApplicationMain
14 OurApp                       0x100937dc4
15 libdyld.dylib                  0x2196868e0 start
```
Crash happens at:
```
__38-[FIRInstanceID notifyAPNSTokenIsSet:]_block_invoke + 1016 (FIRInstanceID.m:1016)
```
with error:
> Fatal Exception: NSInvalidArgumentException
*** -[__NSPlaceholderDictionary initWithObjects:forKeys:count:]: attempt to insert nil object from objects[0]

#### Steps to reproduce:

Crash happens at launch

### PR DESCRIPTION
**Pull request by morganchen12**
Fixes #3439.

### CODE REVIEW

File: Firebase/InstanceID/FIRInstanceID.m [hunk 1]
```diff
@@ -1012,6 +1012,16 @@ - (void)notifyAPNSTokenIsSet:(NSNotification *)notification {
     FIRInstanceID_WEAKIFY(self);
     [self asyncLoadKeyPairWithHandler:^(FIRInstanceIDKeyPair *keyPair, NSError *error) {
       FIRInstanceID_STRONGIFY(self);
+      if (self == nil) {
+        FIRInstanceIDLoggerDebug(kFIRInstanceIDMessageCodeInstanceID017,
```
Comment (chliangGoogle) [inline Firebase/InstanceID/FIRInstanceID.m]: Should this be warning/error level at least? ( same as below)

**Comment by paulb777** [comment]
Fix style and run travis

File: Firebase/InstanceID/FIRInstanceID.m (hunk 1 above)
Comment (morganchen12) [reply Firebase/InstanceID/FIRInstanceID.m]: Done.

**Comment by paulb777** [comment]
@morganchen12 @chliangGoogle @davidair It looks like this missed the 6.6.0 branch by 20 minutes. Should it be added?

**Comment by chliangGoogle** [comment]
I'm fine either way, @morganchen12 can you help release IID if this goes into 6.6.0?

**Comment by davidair** [comment]
We can cherrypick this on Thursday

**Comment by morganchen12** [comment]
I can help release IID. I'm also ok with waiting for the next release if cherry-picking causes significant issues.

**Comment by paulb777** [comment]
I just rebased release-6.6.0 with this fix.

**Comment by paulb777** [comment]
@morganchen12 Please update the IID release note appropriately.

**Pull request merged**